src/Microsoft.DocAsCode.Build.TableOfContents/TocDocumentProcessor.cs

```diff
@@ -169,7 +170,7 @@ private void RegisterTocMap(TocItemViewModel item, string key, IDocumentBuildCon
             }
         }
 
-        private string ResolveHref(string pathToFile, string originalPathToFile, FileModel model, IDocumentBuildContext context)
```

Comment:
I can only see error message change here. How does it fix duplicate warning message issue?